modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/snapshot/SnapshotService.java

```diff
@@ -39,10 +39,7 @@ public void addSnapshotable(Snapshotable snapshotable) {
 
     public byte[] snapshot() {
         HashMap<String, Object[]> snapshots = new HashMap<String, Object[]>(snapshotableList.size());
-
-        if (log.isDebugEnabled()) {
-            log.debug("Taking snapshot ...");
-        }
+        log.info("Taking snapshot ...");
```

Comment:
@ramindu90 

Should these be info logs? Usually we keep the log level to info by default for our carbon servers and taking a single snapshot will produce 3 log entries (with only 'Taking snapshot finished' being set as debug). Won't that create too many log entries in the console? User might be more comfortable seeing a single message saying the synching process was completed successfully or similar. WDYT?